### PR TITLE
Add Cybersecurity Best Practices

### PR DESCRIPTION
**Pull request by sgrest**
New cybersecurity contents are added to this page to provide the clients with security measures that they need to apply or implement when installing or developing using Bonita.

### CODE REVIEW

**Comment by educhastenier** [comment]
test this please

**Comment by educhastenier** [comment]
retest this please

**Comment by educhastenier** [comment]
retest this please

**Comment by educhastenier** [comment]
retest this please

File: md/first-steps-after-setup.md [hunk 1]
```diff
@@ -12,11 +12,43 @@ In the URL field of your web browser, enter the local host address, e.g. `http:/
...
+
+#### Choose a strong password for your connectors
+In Bonita, connectors enable connections to your local information system or online services. The authentication credentials used for these connections are of great value to the business and as a result, they need to be protected. It is highly recommended to choose strong, random and long passwords, as explained above, to makes it difficult or even improbable for one to guess the password. 
+
+#### Prohibit or limit the usage of Java Client
+The Java Client is a tool that can be downloaded and installed for configuration purposes by the administrators. If such tool is of no use for your application, it is highly recommended to prohibit the usage of Java Client in your deployment environment. Otherwise, its usage must be limited to administrators using a local connection or authorized IP addresses.
```
Comment (educhastenier) [inline md/first-steps-after-setup.md]: What do you mean with Java Client? How can you prohibit its usage?
Don't you rather mean to deactivate the Engine HTTP API?

Comment (sgrest) [reply md/first-steps-after-setup.md]: Thanks for your comment. I didn't know that  Engine HTTP API can be deactivated in configuration files. I'll change and reformulate this part.

Comment (tbouffard) [reply md/first-steps-after-setup.md]: See https://documentation.bonitasoft.com/bonita/7.3/configure-client-of-bonita-bpm-engine#toc5 for server side configuration

**Comment by tbouffard** [comment]
@sgrest sorry for the so late review, I haven't seen your new commit. This looks good to me now
It's seem to me that this also applies to the 7.3 version, I am checking this right now

**Comment by tbouffard** [comment]
I have just review the content of this PR and all pages reference here exist in 7.3.
So I am rebasing the PR on the 7.3 branch

**Comment by tbouffard** [comment]
Finally, too much changes between 7.3 and 7.4, so I am merging this PR in the 7.4 branch

**Pull request merged**